test/suite/test_backup01.py

```diff
@@ -163,8 +163,8 @@ def test_cursor_reset(self):
         self.assertEqual(ret, wiredtiger.WT_NOTFOUND)
         self.assertEqual(i, total)
 
-    # Test that named checkpoints can't be deleted while backup cursors are
-    # open, but that normal checkpoints continue to work.
+    # Test that named checkpoints created before a backup cursor can't be deleted while
+    # the backup cursor is open, but that normal checkpoints continue to work.
```

Comment:
Is there any addition that could go here? Such as adding here that named checkpoints created after the backup cursor is opened (or > 1 second after) can be deleted.

Reply:
I thought about adding that, along with a test verifying that we can delete a named checkpoint after a backup.  But @keithbostic's [comment](https://jira.mongodb.org/browse/WT-5242?focusedCommentId=3054288&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-3054288) on not documenting this behavior made me reconsider. If we don't make this promise to users is it important to verify it in the test?

Reply:
These tests are for us. I think it would be easy to add and drop a named checkpoint and make sure it went away as expected. If it behaves unexpectedly we'd want to know regardless of documentation. If the testing gets complicated we can revisit. But if it's easy there's no reason not to do it.

Reply:
Done.  I also added a test to make sure that the named checkpoint that is pinned by the backup cursor can be dropped after the backup cursor is closed.